src/Morphir/Elm/Frontend/Resolve.elm

```diff
@@ -207,6 +207,7 @@ defaultImports =
     , Import (en [ "String" ]) Nothing (Just (en (Explicit [ en (TypeOrAliasExpose "String") ])))
     , Import (en [ "Char" ]) Nothing (Just (en (Explicit [ en (TypeOrAliasExpose "Char") ])))
     , Import (en [ "Tuple" ]) Nothing Nothing
+    , Import (en [ "Rule" ]) Nothing (Just (en (Explicit [ en (TypeOrAliasExpose "Rule") ])))
```

Comment:
This should mirror the default imports of Elm. Can you remove this entry since Rule is not auto-imported by Elm so we will have an inconsistency.